packages/devkit/src/generators/project-configuration.ts

```diff
@@ -398,24 +454,53 @@ function getProjectFileLocation(tree: Tree, project: string): string | null {
     : null;
 }
 
-function validateWorkspaceJsonOperations(
+function validateProjectConfigurationOperations(
   mode: 'create' | 'update' | 'delete',
-  workspaceJson: RawWorkspaceJsonConfiguration | WorkspaceJsonConfiguration,
-  projectName: string
+  workspaceJson:
+    | RawWorkspaceJsonConfiguration
+    | WorkspaceJsonConfiguration
+    | null,
+  projectName: string,
```

Comment:
For cases where we have no worskpace.json, we need to be able to check files existence at project root, so this f(x) requires a bit more args now.